src/register/ThirdPartyAuth.jsx

```diff
@@ -24,6 +24,7 @@ const ThirdPartyAuth = (props) => {
   const isInstitutionAuthActive = !!secondaryProviders.length && !currentProvider;
   const isSocialAuthActive = !!providers.length && !currentProvider;
   const isEnterpriseLoginDisabled = getConfig().DISABLE_ENTERPRISE_LOGIN;
+  const ThirdPartyAuthPreloader = isSocialAuthActive || (isEnterpriseLoginDisabled && isInstitutionAuthActive);
```

Comment:
[Same concern again] I believe ThirdPartyAuthPreloader is supposed to be a boolean flag. Could you please refactor it to thirdPartyAuthPreloader

Reply:
Changed to isThirdPartyAuthActive